src/gui/sdlmain.cpp

```diff
@@ -1313,30 +1322,50 @@ static void ToggleMouseCapture(bool pressed) {
 	GFX_ToggleMouseCapture();
 }
 
-void GFX_UpdateMouseAfterExposure(void) {
+/*  
+ *  Assesses the following:
+ *   - current window size (full or not),
+ *   - mouse capture state, (yes or no).
+ *   - desired capture type (start, click, seamless), and
+ *   - if we're starting up for the first time,
+ *  to determine if the mouse-capture state should be toggled.
+ *  Note that this also acts a filter: we don't want to repeatedly
+ *  re-apply the same mouse capture state over and over again, so most
+ *  of the time this function will (or should) decide to do nothing.
+ */
+void GFX_UpdateMouseState(void) {
```

Comment:
This is a tangent, and not exactly related to this code, so TL;DR is: let's reformat this as
```
void GFX_UpdateMouseState()
{
```
as this is C++ and not C, so we don't need to care about C89-isms ;)

### Longer explanation

Some time ago I wondered why so many programmers place `void` for function parameters, and as it turned out this is cruft of old C function declaration syntax (C89 and older). Practically nobody is using that old C syntax any more, here's an example of [how it changed](https://www.quora.com/How-have-the-function-declarations-changed-over-time-in-various-C-version-ANSI-C-C89-C95-C99-and-C11). Old syntax is not supported in C++ any more (last time it was supported was in C++85, I think) - so `int f(void);` and `int f();` mean exactly the same in C++ for more than 30 years now. In modern C old syntax it's still supported, but the C steering committee plans to drop it eventually.

I think the only reason why some programmers still use it is because they learned to program using Visual Studio in 90s (MSVC supported only C89 for looong time) and now they propagate it in cargo-cult manner.

So let's just stop preserving `void` to mean "this function takes no parameters" - it's redundant :).

Reply:
I'm also surprise I didn't clean up that up :-)  This next commit mass-cleans them up in the mapper.